NodeJS-home-task-2/routes/routers.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const uuid = require("uuid");
+const userSchema = require("../model/users.js").userSchema;
+const router = express.Router();
+
+const users = [];                                    //for in-memory
+
+//for validating according to created schema
+const userValidator = (schema) => {
+    return (req, res, next) => {
+        const { error } = schema.validate(req.body);
+        if (error === null || error === void 0 ? void 0 : error.isJoi) {
+            res.json(error.message);
+        }
+        else {
+            next();
+        }
+    };
+};
+
+//get all users
+router.get('/users', (req, res) => {
+    const loginSubstring = '', limit = 10;
+    const list = users
```

Comment:
lets put this logic in another file. May be service.
It should call 
const result = await userService.getUserById(id);
keep the status code in this file only, just move the business logic to another file

Reply:
Ignore, you have made changes in task 3